packages/material-ui-lab/src/TreeItem/TreeItem.d.ts

```diff
@@ -38,7 +38,6 @@ export interface TreeItemProps
   collapseIcon?: React.ReactNode;
   /**
    * The component used for the content node.
-   * @default TreeItemContent
    */
```

Comment:
@mnajdova I don't want to remove this but it failed at `yarn docs:api`. What should be the fix?

Reply:
It is because you actually remove the default values in the implementation - https://github.com/mui-org/material-ui/pull/25835/files#diff-0bcf76be35b3a4a1283be7fd0b9152e7c794573aa81068fa11d3c1436d8619a8L102

Reply:
Also we should add the missing types for the `sx` prop here right?